invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Headline/Headline.js

```diff
@@ -64,8 +64,7 @@ export default class Headline extends Component {
                   className={'headline-quick-access'}
                   as={Link}
                   to={FrontSiteRoutes.documentsListWithQuery(
-                    'document_type:BOOK&sort=mostrecent&order=desc' +
-                      '&aggr[0][has_eitems][value]=has_eitems.electronic versions'
+                    '&f=doctype%3ABOOK&f=medium%3AELECTRONIC_VERSION&sort=mostrecent&order=desc'
```

Comment:
thanks, I forgot about this, and it is a bit dangerous to hardcode the queries. I will start a ticket to clean this, maybe we will have a nice idea, maybe as a part of query builder